hack/github_release_run.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
+
+# todo: replace release id with /latest after non-pre-release exists
+# REL_URL=https://api.github.com/repos/oneconcern/datamon/releases/latest
+REL_URL=https://api.github.com/repos/oneconcern/datamon/releases/16541641
```

Comment:
We will cut a release soon that works with latest, you can flip this to latest. Will merge it in and cut the release in this week.